Add files via upload

frankylox opened the pull request:
Данная игра запускается в любом IDE с версией Python 3+ (наверное). 
Имеется счетчик очков, который увеличивается посредством нажатия на пробел. За 100 кликов можно купить улучшение, которое увеличивает количество очков per клик на один.

frankylox committed:
Add files via upload

febos reviewed (commented):
1) Зажатие клавиши накручивает очки - не тру

2) простоватенько

3) у вас неделя на доделки

frankylox committed:
Update main_clicker.py

все еще простоватенько, но хотя бы работает ㅠㅠ

frankylox committed:
Update main_clicker.py x2

Вместо миллиона лишних нажатий он добавляет всего одно, но вроде как в оригинальном куки кликере такой баг тоже был
